linux_install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# This is the install script for installing vs code onto a linux machine
+# Usage: sudo bash linux_install.sh 
+userid=`id -u`
+if [[ $userid -ne 0 ]]
+then
+   echo "Sorry, you aren't a root user"
+fi
+
+cmd=`mkdir "/usr/local/vscode/"`
+
+if [[ $cmd -eq 0 ]]; then
+	echo "folder creation successful"
+else
+	echo "something went wrong creating folder"
+fi
+
+cmd=`cp vscode.desktop /usr/share/applications`
+
+if [[ $cmd -eq 0 ]]; then
+	echo "successfully installed app shortcut"
+else
+	echo "something went wrong with installing app shortcut"
+fi
+
+cmd=`cp -r * /usr/local/vscode/`
+
+if [[ $cmd -eq 0 ]]; then
+	echo "successfully installed app"
+else
+	echo "something went wrong with installing app"
+fi
+
+echo 'export PATH=$PATH:/usr/local/vscode' >> ~/.bashrc
```

Comment:
no the bashrc isn't the root bashrc, it is the user's bashrc. I realize the folly now, there is no need to have vscode in the PATH variable! And no, it isn't tailored for one user only, it is tailored for my user on my laptop :smile: I am a bit new to all this, do you know of any install script which is standard and does all the things which you said, like the undo command and all?